quoridor-app/app/api/gameApi.js

```diff
@@ -30,3 +30,24 @@ export const joinGame = async gameId => {
     }
   });
 };
+
+export const movePawn = async (authToken, gameId, position) => {
```

Comment:
It's not necessary to have an async function here. You don't wait for the return of the fetch, so you can just return your Promise.